Added initial English localization for documentation (#13)
furiosa-ai/furiosa-sdk: docs/en/source/installation/runtime.rst

**********************************
NPU Runtime Installation
**********************************

> FuriosaAI NPU Runtime includes the NPU runtime for accelerating a deep learning model on the NPU,

Reviewer: ```suggestion
FuriosaAI NPU Runtime includes the NPU runtime for accelerating deep learning models on the NPU,
```

> an interface for executing it, and various tools required for NPU application development.

Reviewer: ```suggestion
inference API, and various tools required for NPU application development.
```

> FuriosaAI NPU Runtime can be used by installing the following two packages, while the deb package 

Reviewer: ```suggestion
FuriosaAI NPU Runtime can be used by installing the following two packages, while the deb packages
```

> configuration is delivered through a separate channel after the accepting the End User License Agreement 

Reviewer: ```suggestion
will be delivered through a separate channel after the accepting the End User License Agreement 
```
